tasks/main.yml

```diff
@@ -55,10 +66,14 @@
     password: "{{ item.value.password }}"
     role_attr_flags: "{{ item.value.roles | default(postgres_default_roles) | join(',') }}"
     state: present
+  vars:
+    ansible_ssh_pipelining: true
   with_dict: "{{ postgres_users }}"
+  tags: postgres-create-users
 
 - name: enable postgres
   systemd:
     enabled: yes
     name: postgresql.service
     state: started
+  tags: travis-exempt
```

Comment:
Should be generalized to systems that share similar system constraints as mentioned above.